lib/Experiment.js

```diff
@@ -142,18 +149,26 @@ class Experiment {
 		desc += `${tabs}\tVariations:\n`
 		this[_variations].forEach((variation, index) => {
 			let isBase = '' //index === 0 ? '<== base variation' : ''
-			desc += `${tabs}\t\t#${index} ${variation.toString(`${tabs}\t\t`)} ${isBase}\n`
+			desc += `${tabs}\t\t#${index} ${variation.toString(
```

Comment:
this is bad also... but to be honest it wasn't great to begin with! :)

Reply:
Again this is because max line width is 80.
But in this case I think prettier is trying to tell us to extract `variation.toString(`${tabs}\t\t`)` into a variable.

Reply:
can we make prettier max line width longer? the 80 chars limits is something from days of antiquity...?